Maths/ExponentialFunction.js

```diff
@@ -10,16 +8,21 @@ import { calcFactorial } from './Factorial.js'
  */
 function exponentialFunction (power, n) {
   let output = 0
+  let fac = 1
   if (isNaN(power) || isNaN(n) || n < 0) {
     throw new TypeError('Invalid Input')
   }
   if (n === 0) { return 1 }
   for (let i = 0; i < n; i++) {
-    output += (power ** i) / calcFactorial(i)
+    output += (power ** i) / fac
+    fac *= ++i
```

Comment:
Either iterate `i` from `1` to `n` or use `i + 1` here. Incrementing & then decrementing `i` is dirty.